test/readdlm.jl

```diff
@@ -162,7 +162,7 @@ let i18n_data = ["Origin (English)", "Name (English)", "Origin (Native)", "Name
         "Ireland", "Sinead O'Connor", "Éire", "Sinéad O'Connor",
         "Israel", "Yehoram Gaon", "ישראל", "יהורם גאון",
         "Italy", "Fabrizio DeAndre", "Italia", "Fabrizio De André",
-        "Japan", "KUBOTA Toshinobu", "日本", "久保田    利伸",
+        "Japan", "KUBOTA Toshinobu", "日本", "久保田    利伸",
```

Comment:
I noticed that and run the test locally to make sure it passes.

At a glance, I think this code is writing a array to a csv file and check if it can real the same back. So changing the data won't break anything